ceph_deploy/hosts/debian/mon/create.py

```diff
@@ -1,42 +1,56 @@
 from ceph_deploy.hosts import common
+from ceph_deploy.util import system
 from ceph_deploy.lib import remoto
 
 
 def create(distro, args, monitor_keyring):
-    logger = distro.conn.logger
     hostname = distro.conn.remote_module.shortname()
     common.mon_create(distro, args, monitor_keyring, hostname)
-    service = distro.conn.remote_module.which_service()
 
-    if not service:
-        logger.warning('could not find `service` executable')
-
-    if distro.init == 'upstart':  # Ubuntu uses upstart
```

Comment:
The sysvinit and systemd stuff look fine, but unless my eyes are playing tricks upstart got removed entirely.  So this would break on 14.04, for example.